test/tests/functional/pbs_reservations.py

```diff
@@ -2310,3 +2310,177 @@ def test_resv_reconfirm_holding_partial_nodes(self):
         solution = '(' + vn[0] + ':ncpus=2)+(' + resv_node2 + ':ncpus=1)'
         a = {'reserve_substate': '5', 'resv_nodes': solution}
         self.server.expect(RESV, a, id=rid)
+
+    @skipOnCpuSet
+    def test_qmove_job_into_standing_reservation(self):
+        """
+        Test qmove job into standing reservation
+        """
+        a = {'resources_available.ncpus': 2}
+        self.server.manager(MGR_CMD_SET, NODE, a, id=self.mom.shortname)
+        j1 = Job(attrs={'Resource_List.walltime': 100})
+        jid1 = self.server.submit(j1)
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid1)
+
+        now = int(time.time())
+        # Submit a standing reservation to occur every other minute for a
+        # total count of 2
+        start = time.time() + 10
+        now = start + 50
+        start = int(start)
+        end = int(now)
+        rid = self.submit_reservation(user=TEST_USER, select='1:ncpus=1',
+                                      rrule='FREQ=MINUTELY;INTERVAL=2;COUNT=2',
+                                      start=start, end=end)
+        rid_q = rid.split('.')[0]
+        exp_attr = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2")}
+        self.server.expect(RESV, exp_attr, id=rid)
```

Comment:
@prakashcv13 By the time job is moved into resv, the resv might be in running state. This would be the same scenario tested in "test_qmove_job_into_standing_reservation_case2". Increasing the resv start time would also suffice instead of my above suggestion.

Reply:
your comment is about how the overall test case should be written, but you have commented in the place where we check for the reservation being confirmed, hence the confusion.